lte/gateway/python/magma/pipelined/ng_set_session_msg.py

```diff
@@ -190,7 +198,7 @@ def CreateSession(
         self, imsi_val: str, pdr_state: str = "ADD", in_teid: int = 0,
         out_teid: int = 0, ue_ip_addr: str = "", gnb_ip_addr: str = "",
         del_rule_id: str = '', add_rule_id: str = '', ipv4_dst: str = None, allow: str = "YES",
-        priority: int = 10, hard_timeout: int = 0,
+        priority: int = 10, hard_timeout: int = 0, apn_flag: bool = False,
```

Comment:
This bool seems odd to me, just pass apn_ambr directly, with a None default

Reply:
We tried passing apn_ambr but its always taking it None regardless of the QoS flag maybe because its a structured message of AggregatedMaximumBitrate type. That's why we introduced this apn_flag, boolean flag.

Reply:
hm I'm not sure if I understand.
This signature
```

    def CreateSession(
        self, imsi_val: str, pdr_state: str = "ADD", in_teid: int = 0,
        out_teid: int = 0, ue_ip_addr: str = "", gnb_ip_addr: str = "",
        del_rule_id: str = '', add_rule_id: str = '', ipv4_dst: str = None, allow: str = "YES",
        priority: int = 10, hard_timeout: int = 0,
        priority: int = 10, hard_timeout: int = 0, apn_ambr: AggregatedMaximumBitrate = None,
```
With this usage should work
```
    if args.disable_qos:
        print("QOS Disabled")
        apn_ambr = None
    else:
        print("QOS Enabled")
        apn_ambr = AggregatedMaximumBitrate(
                max_bandwidth_ul=1000000000,
                max_bandwidth_dl=1000000000,
            )
```

Reply:
Fixed